examples/zenohcxx/zenohc/z_get_channel.cxx

```diff
@@ -17,9 +17,8 @@
 #include <condition_variable>
 #include <iostream>
 
-#include "zenohcpp.h"
-
-using namespace zenoh;
+#include "zenohc.hxx"
+using namespace zenohc;
```

Comment:
Shouldn't it be `zenoh` namespace like in the other examples?

Reply:
No, this is specific zenohc example and this demonstrates how to directly use zenohcxx for zenohc. This is also mentioned in readme